pkg/conthandler/container_main_handler.go

```diff
@@ -0,0 +1,5 @@
+package conthandler
+
+type ContainerMainHandlerClient interface {
+	StartMainHandler() error
```

Comment:
[TODO]: I dont see a reason of wrapping the main function in an interface. What other options are there to run here?